src/connectedvmware/azext_connectedvmware/tests/latest/test_arcvmware_scenario.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+from knack.util import CLIError
+from azure.cli.testsdk import ScenarioTest
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class ConnectedvmwareScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_connectedvmware')
+    def test_connectedvmware_vcenter(self, resource_group):
+
+        self.kwargs.update({'name': 'test1'})
+
+        # raise CLIError('TODO')
```

Comment:
Why is there nothing in this test? CI failed because it did not record a yaml file

Reply:
Earlier our service extension name was "arcvmware" then we have changed it to "connectedvmware" now all the test has been written in "test_connectedvmware_scenario.py" file. "test_arcvmware_scenario.py" is just left over I have removed this in latest push.